modules/gdscript/gdscript_editor.cpp

```diff
@@ -1048,10 +1048,19 @@ static void _find_global_enums(HashMap<String, ScriptLanguage::CodeCompletionOpt
 	}
 }
 
+static void _add_variant_types(HashMap<String, ScriptLanguage::CodeCompletionOption> &r_result) {
+	ScriptLanguage::CodeCompletionOption variant_option("Variant", ScriptLanguage::CODE_COMPLETION_KIND_CLASS);
+	r_result.insert(variant_option.display, variant_option);
+}
```

Comment:
```suggestion
}

```
Please keep an empty line between methods